### ISSUE TITLE
Specify "play area" and "control area" for each screen

### ISSUE THREAD

**Issue by pixelzoom** [phetsims/fourier-making-waves#76]
From https://github.com/phetsims/fourier-making-waves/issues/53#issuecomment-856099325:

- [ ] @arouinfar will make a first pass at splitting traversal into "play area" and "control area". Even though we had considered not doing this for 1.0, the implementation of ScreenView makes it clear that the intent in to add Nodes to playArea or controlArea, and NOT to the ScreenView. This is going to be tricky for Wave Game screen, related to phetsims/vegas#90.

See https://github.com/phetsims/vegas/issues/93 for discussion about the "tricky" part of Wave Game screen, and game screens in general.

The specification should be documented in https://docs.google.com/document/d/1wOdmPMD704u4OLl9avI9tl2jpTvsAAcZwMaNfnr0qSs/edit?pli=1#heading=h.qbkcpoz1vzlo

**Comment by pixelzoom** [comment]
6/10/21 design meeting: @arouinfar @kathy-phet @jessegreenberg @KatieWoe 

- JG says this will be technically more difficult than accessibleNames, CM agrees (especially Wave Game)
- Whatever we do will need to be revisited for interactive description.
- We could put just TimeControlNode and ResetAllButton in controlArea for Discrete and Continuous screen, but that doesn't really give us anything.
- JG felt that this was not that useful for users.
- consensus: skip for 1.0 release

I'm going to close this issue, and this will be addressed sometime in the future, probably when the sim gets an Interactive Descriptions design.
